src/core/thread/mle.cpp

```diff
@@ -1560,6 +1560,7 @@ ThreadError Mle::SendChildUpdateRequest(void)
     VerifyOrExit((message = NewMleMessage()) != NULL, ;);
     SuccessOrExit(error = AppendHeader(*message, Header::kCommandChildUpdateRequest));
     SuccessOrExit(error = AppendMode(*message, mDeviceMode));
+    SuccessOrExit(error = AppendSourceAddress(*message));
```

Comment:
I believe this check was erroneously added in V39 and is tracked in [DEV-1358](https://threadgroup.atlassian.net/browse/DEV-1358?jql=project%20%3D%20DEV%20AND%20text%20~%20%22Source%20Address%20TLV%22).  @librasungirl, can you confirm?